src/Altinn.Platform/Altinn.Platform.Storage/Storage/Program.cs

```diff
@@ -57,15 +61,12 @@ public static void Main(string[] args)
                                 .CreateLogger();
 
                 logging.AddProvider(new SerilogLoggerProvider(logger));
-            })
-
-            /* // Parameters required for integration with SBL in local development             
-             .UseKestrel()
-             .UseUrls("http://0.0.0.0:5010")
-            .UseContentRoot(Directory.GetCurrentDirectory())
-            .UseIISIntegration() */
-            .UseApplicationInsights()
```

Comment:
I cannot see that this is added anywhere else. Did it break the code?

Reply:
+1

Reply:
this is replaced with services.AddApplicationInsightsTelemetry();